### DISCUSSION TITLE
What language is the settings file in?

### DISCUSSION THREAD

**Discussion by God-damnit-all** [sumatrapdfreader/sumatrapdf#3937] [Q&A] [answered]
Just curious what language the settings file is in, or if it's just something specific to SumatraPDF. It's kind of like some sort of ini / yaml hybrid?

**Comment by kjk** [comment, marked as answer]
This is our custom format. And yes, like you've noticed, it's inspired by ini format

**Reply by God-damnit-all** [reply]
Thanks for taking the time to let me know. Does it have a name? It'd be neat if it got ported into its own library at some point.

**Reply by kjk** [reply]
no, I didn't give it a name
